lib/ui/nav_pages/home_page/home_page.dart

```diff
@@ -36,10 +36,10 @@ class HomePage extends StatelessWidget {
           ),
           model.isBusy
               ? LinearProgressIndicator(
-                  backgroundColor: Colors.grey[400],
-                  valueColor:
-                      const AlwaysStoppedAnimation(AppColors.zuriPrimaryColor),
-                )
+            backgroundColor: Colors.grey[400],
```

Comment:
Use the AppColors class

Reply:
i'm not working on home_page, i only resolved conflict in it, so I could push my commits